manifests/init.pp

```diff
@@ -276,6 +277,7 @@
   Stdlib::Absolutepath $inetrc_config_path                         = $rabbitmq::params::inetrc_config_path,
   Boolean $ssl_erl_dist                                            = $rabbitmq::params::ssl_erl_dist,
   Optional[String] $rabbitmqadmin_package                          = $rabbitmq::params::rabbitmqadmin_package,
+  Optional[Array] $mqa_download_opts                               = $rabbitmq::params::mqa_download_opts,
```

Comment:
This should either be set to `undef` and left directly here (vs. coming from params), or shouldn't be optional. Except when using hiera, Puppet won't let you set a default for an optional type for complicated (and historical) reasons, so an Optional param with a default is not a good idea.